cpp/src/DataStorm/NodeSessionManager.cpp

```diff
@@ -59,25 +61,18 @@ NodeSessionManager::NodeSessionManager(const shared_ptr<Instance>& instance, con
           instance->getCommunicator()->getProperties()->getIcePropertyAsInt(
               "DataStorm.Node.Server.ForwardDiscoveryToMulticast") > 0),
       _retryCount(0),
-      _forwarder(instance->getCollocatedForwarder()->add<LookupPrx>([this](Ice::ByteSeq e, const Ice::Current& c)
-                                                                    { forward(e, c); }))
+      _forwarder(instance->getCollocatedForwarder()->add<LookupPrx>(
+          [this](Ice::ByteSeq inParams, const Ice::Current& current) { forward(inParams, current); }))
 {
 }
 
 void
 NodeSessionManager::init()
 {
     auto instance = getInstance();
-
-    try
-    {
-        auto sessionForwader = make_shared<SessionForwarderI>(shared_from_this());
-        instance->getObjectAdapter()->addDefaultServant(sessionForwader, "sf");
-        instance->getObjectAdapter()->addDefaultServant(sessionForwader, "pf");
-    }
-    catch (const Ice::ObjectAdapterDestroyedException&)
```

Comment:
This code runs during Node initialization, no point in handling this and keep going.